packaging/deb/rules

```diff
@@ -32,10 +32,8 @@ override_dh_auto_configure:
 	# nothing to configure
 
 override_dh_auto_build:
-ifeq ($(ARCH),ppc64le)
 	bundle config set --global force_ruby_platform true
```

Comment:
Should this `force_ruby_platform` be removed since also being removed from RPM spec?  Just for consistency sake.

Reply:
It's not being removed - the if block is. I mean, all distributions and architectures are going to `force_ruby_platform`, or at least that's the point of the change.

Reply:
Right, but the RPM spec the `force_ruby_platform` item is removed it appears like from the diff.  So would it make sense to remove in Deb builds too?

Reply:
Ah sorry, the diff threw me off.  They both have `force_ruby_platform`.